witch main

```diff
@@ -0,0 +1,23 @@
+  VFEP-11[m
```

Comment:
Sorry to revive an old PR, but is this file supposed to be in here?

Reply:
Hi @jerekshoe , no it is not. somehow when I was merging main into the branch then trying to run git switch main, it created this file with branch names and named it "witch main". I believe I have it deleted and it shouldn't be in the main branch.

Reply:
The reason I asked was because I pulled main recently and noticed that it was there. I can make a PR to remove if you want, just figured I'd verify that it didn't serve a purpose. https://github.com/department-of-veterans-affairs/vets-website/blob/main/witch%20main

Reply:
Makes sense. Yes, please do remove it. Its just a junk file that has no reason to be in the repo.